.flake8

```diff
@@ -13,6 +13,7 @@ per-file-ignores =
     ./src/openeo_grass_gis_driver/models/job_schemas.py: W605, E501
     ./src/openeo_grass_gis_driver/models/service_schemas.py: W605
     ./src/openeo_grass_gis_driver/collection_information.py: E501
+    ./src/openeo_grass_gis_driver/local_collections.py: E501
```

Comment:
Sorry, made a mistake in my last comments. `collection_information.py` needs to stay but the new one should be valid.
```suggestion
```